library/Fission/Mail/SendInBlue/TemplateId/Types.hs

```diff
@@ -0,0 +1,10 @@
+module Fission.Mail.SendInBlue.TemplateId.Types (TemplateId(..)) where
+
+import Fission.Prelude
+
+newtype TemplateId = TemplateId { unTemplateId :: Int64 }
+  deriving          ( Generic )
```

Comment:
🔪 `Generic`

Reply:
oh really? when do we want `Generic` I thought we derived it anytime the type was a simple wrapper of some primitive

Reply:
> we derived it anytime the type was a simple wrapper of some primitive

Nope! It's very possible (and common!) to have a `Generic`-free codebase. Generic is only used by the compiler to help derive certain types of instances automatically (e.g. `ToParamSchema`). Since we're opting to make these explicit, we don't need the compiler to do the work of creating a bunch of extra compile-time annotations.